x/qgb/keeper/keeper_attestation.go

```diff
@@ -78,6 +78,36 @@ func (k Keeper) GetLatestAttestationNonce(ctx sdk.Context) uint64 {
 	return UInt64FromBytes(bytes)
 }
 
+// CheckEarliestAvailableAttestationNonce returns true if the last available attestation nonce has been initialized
+// in store, and false if not.
+func (k Keeper) CheckEarliestAvailableAttestationNonce(ctx sdk.Context) bool {
+	store := ctx.KVStore(k.storeKey)
+	has := store.Has([]byte(types.LatestAttestationtNonce))
+	return has
+}
+
+// GetEarliestAvailableAttestationNonce returns the earliest available attestation nonce. The
+// nonce is of the last available attestation in store that can be
+// retrieved. Panics if the last available attestation
+// nonce doesn't exit. This value is set on chain startup. However, it won't be
+// written to store until height = 1. Thus, it's mandatory to run
+// `CheckEarliestAvailableAttestationNonce` before calling this method.
```

Comment:
[uber nit][not blocking] the comments in this PR are wrapped at arbitrary line lengths. https://marketplace.visualstudio.com/items?itemName=stkb.rewrap will rewrap comments to a fixed length so that they appear cleaner.

Reply:
https://github.com/celestiaorg/celestia-app/issues/1819